src/blocks/subscribe/index.php

```diff
@@ -241,6 +248,15 @@ function process_form() {
 		return send_form_response( [ 'email' => \sanitize_email( $_REQUEST['email'] ) ] );
 	}
 
+	// reCAPTCHA test.
+	if ( ! empty( $_REQUEST['captcha_token'] ) && method_exists( '\Newspack\Reader_Activation', 'verify_captcha' ) ) {
```

Comment:
This check allows for the captcha test to be skipped if not sent. It must fail if it's enabled and the request doesn't provide a token.

Reply:
Updated in 37f658